decidim-budgets/db/migrate/20170612101846_migrate_projects_category.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class MigrateProjectsCategory < ActiveRecord::Migration[5.1]
+  def change
+    Decidim::Budgets::Project.find_each do |project|
+      Decidim::Categorization.create!(
+        decidim_category_id: project.category.id,
```

Comment:
The point of using SQL in migrations is that you will now that the database is consistent at that point of time. Models can be wrong about their assumptions. So unless there's a compelling reason, SQL by default plzzzzzzz.